src/utils.js

```diff
@@ -0,0 +1,14 @@
+import Transaction from 'arweave/node/lib/transaction';
+
+
+export function interactionTagsParser(tx) {
+    const contractTx = new Transaction(tx.contractTx);
```

Comment:
you should pass here the tx and not 'data' as you are currently doing in `Contract.vue` page. 
So pass there `data.contractTx` and here just receive the tx.

Reply:
done